docs/source-app/conf.py

```diff
@@ -149,8 +149,8 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "pt_lightning_sphinx_theme"
-html_theme_path = [pt_lightning_sphinx_theme.get_html_theme_path()]
+html_theme = "lai_sphinx_theme"
+html_theme_path = [lai_sphinx_theme.get_html_theme_path()]
```

Comment:
There were other places where this path is hardcoded in `docs4`. Let's make sure we replace hardcoded paths with `html_theme_path = [lai_sphinx_theme.get_html_theme_path()]` before merging.

Reply:
yes, but this is not using #15473